src/file/file.ts

```diff
@@ -85,13 +85,13 @@ function findNextFileNumber(folderPath: string, vault: Vault) {
   return nextNumber === 0 ? "" : nextNumber.toString();
 }
 
-function getNewFileContent(template: Template): string {
+function getNewFileContent(template: Template, hashtag: string): string {
   switch (template) {
     case Template.CUSTOM:
-      return customFormat;
+      return hashtag + '\n' + customFormat;
```

Comment:
If hashtag is empty (by default), an additional newline character will be added.

Reply:
Thank you, fixed this!